### PR TITLE
Fixes #43

### LINKED ISSUE #43: VersionHandler not working on machines that use comma separation instead of the dot
Basically, `float.TryParse()` will return false if the OS is in a language that uses comma as the decimal separator instead of the dot. This means that it will fall back to 5.4, which in turn causes it to attempt to load BuildTarget number 11, resulting in an instant crash.

### PR DESCRIPTION
**Pull request by bence98**
I have already signed the CLA.

### CODE REVIEW

**Comment by stewartmiles** [comment]
Thanks for the patch!

File: source/VersionHandler/src/VersionHandler.cs [hunk 1]
```diff
@@ -165,7 +166,7 @@ private class FilenameComponents
             if (!string.IsNullOrEmpty(version)) {
                 int dotIndex = version.IndexOf('.');
                 if (dotIndex > 0 && version.Length > dotIndex + 1) {
-                    if (!float.TryParse(version.Substring(0, dotIndex + 2), out result)) {
+                    if (!float.TryParse(version.Substring(0, dotIndex + 2), NumberStyles.Any, CultureInfo.InvariantCulture, out result)) {
```
Comment (bence98) [inline source/VersionHandler/src/VersionHandler.cs]: So, `float.TryParse()` is locale-sensitive by default. Thus, `float.TryParse("5.5"; out result)` will return true on an English machine, but false on a German one.

Comment (bence98) [inline source/VersionHandler/src/VersionHandler.cs]: But Unity's version will always be in format "X.Y", which is not a valid float on German/Russian/etc. OSes. Therefore, `float.TryParse()` should ignore the locale.

**Comment by bence98** [comment]
Will this be fixed then?
(Sorry if I sound too impatient, I'm just anxious about a project I have to release by the 28th, and this bug insta-crashes Unity 100% of the time, so I have to use another, poorly performing computer with an English OS)

**Comment by bence98** [comment]
(Plus, I expect some delay between this bug getting fixed in JarResolver, and the AdMobile guys releasing a new plugin version, compiled against the fixed JarResolver)

**Comment by stewartmiles** [comment]
Ok, gotcha.  I'll merge this now.   In terms of release delay I'm not sure what the schedule is @rampara do you have a sense of when you could merge this patch?

**Comment by bence98** [comment]
Thank you! :)

**Comment by rampara** [comment]
@bence98 I'm not sure I understand why you need a new version of the AdMob plugin. Doesn't building JarResolver with this merged fix and importing that into your Unity project resolve your issue? 

The [GMA Unity Gradle build script](https://github.com/googleads/googleads-mobile-unity/blob/master/build.gradle) does pull in the latest version of the Unity package from the root directory of this repo. Once that package is updated, you can build a new version of the GMA plugin that includes this fix.

**Comment by bence98** [comment]
@rampara I am bad with Gradle, so I ended up importing the stock GMA 3.4 unitypackage, waited for Unity to crash&close, deleted PlayServicesResolver folder and imported PlayServicesResolver 1.2.16. It works like a charm, just was tedious. Thank you both for your time!

**Pull request merged**